packages/tools/examples/videoSplineROITools/index.ts

```diff
@@ -0,0 +1,318 @@
+  },
+  BSplineROI: {
+    splineType: SplineROITool.SplineTypes.BSpline,
+  },
+};
+
+const SplineToolNames = Object.keys(Splines);
+const toolsNames = [...SplineToolNames, LivewireContourTool.toolName];
+let selectedToolName = toolsNames[0];
+
+addDropdownToToolbar({
+  options: { values: toolsNames, defaultValue: selectedToolName },
+  onSelectedValueChange: (newSelectedToolNameAsStringOrNumber) => {
+    const newSelectedToolName = String(newSelectedToolNameAsStringOrNumber);
+    const toolGroup = ToolGroupManager.getToolGroup(toolGroupId);
+
+    // Set the old tool passive
+    toolGroup.setToolPassive(selectedToolName);
+
+    // Set the new tool active
+    toolGroup.setToolActive(newSelectedToolName, {
+      bindings: [
+        {
+          mouseButton: MouseBindings.Primary, // Left Click
+        },
+      ],
+    });
+
+    selectedToolName = <string>newSelectedToolName;
+
+    const resolutionSlider = document.getElementById(
+      'splineResolution'
+    ) as HTMLInputElement;
+
+    const scaleSlider = document.getElementById(
+      'splineScale'
+    ) as HTMLInputElement;
+
+    resolutionSlider.disabled = selectedToolName === 'LinearSplineROI';
+    scaleSlider.disabled = selectedToolName !== 'CardinalSplineROI';
+  },
+});
+
+addCheckboxToToolbar({
+  title: 'Preview',
+  checked: true,
+  onChange: (drawPreviewEnabled) => {
+    const toolGroup = ToolGroupManager.getToolGroup(toolGroupId);
+
+    SplineToolNames.forEach((splineToolName) => {
+      const splineConfig = toolGroup.getToolConfiguration(
+        splineToolName,
+        'spline'
+      );
+      splineConfig.drawPreviewEnabled = drawPreviewEnabled;
+      toolGroup.setToolConfiguration(splineToolName, { spline: splineConfig });
+    });
+  },
+});
+
+addSliderToToolbar({
+  id: 'splineResolution',
+  title: 'Resolution',
+  range: [0, 32],
+  step: 1,
+  defaultValue: DEFAULT_SPLINE_RESOLUTION,
+  onSelectedValueChange: (value: string) => {
+    const toolGroup = ToolGroupManager.getToolGroup(toolGroupId);
+
+    SplineToolNames.forEach((splineToolName) => {
+      const { splineType } = Splines[selectedToolName];
+      const splineConfig = toolGroup.getToolConfiguration(
+        splineToolName,
+        'spline'
+      );
+
+      splineConfig.configuration[splineType].resolution = value;
+      toolGroup.setToolConfiguration(splineToolName, { spline: splineConfig });
+    });
+  },
+});
+
+addSliderToToolbar({
+  id: 'splineScale',
+  title: 'Scale',
+  range: [0, 1],
+  step: 0.1,
+  defaultValue: DEFAULT_CARDINAL_SCALE,
+  onSelectedValueChange: (value: string) => {
+    const toolGroup = ToolGroupManager.getToolGroup(toolGroupId);
+    const splineToolName = 'CardinalSplineROI';
+
+    const splineConfig = toolGroup.getToolConfiguration(
+      splineToolName,
+      'spline'
+    );
+
+    splineConfig.configuration.CARDINAL.scale = value;
+    toolGroup.setToolConfiguration(splineToolName, { spline: splineConfig });
+  },
+});
+
```

Comment:
I think these are not necessary for the sake of this example, user can explore the spline settings in the spline example, or maybe use one only

Reply:
I could also add the spline tool to the videoTools example, and just let it be there, without further embellishment?